server/server.go

```diff
@@ -241,3 +242,17 @@ func (srv *Server) UnbindHeartbeatStream(_ context.Context, node string) error {
 func (srv *Server) ProcessHeartbeat(ctx context.Context, req *metaservicepb.NodeHeartbeatRequest) error {
 	return srv.clusterManager.RegisterNode(ctx, req.GetHeader().GetClusterName(), req.GetInfo().GetNode(), req.GetInfo().GetLease())
 }
+
+func (srv *Server) AfterElected(ctx context.Context) {
```

Comment:
It is weird to expose the two methods on `Server`. Shall we define a wrapper for `Server` and implement `LeadershipCallbacks` on the wrapper?